pycheribuild/projects/cross/cheribsd.py

```diff
@@ -1192,6 +1192,13 @@ def __init__(self, config: CheriConfig):
             self.make_args.set(CROSS_BINUTILS_PREFIX=str(self.sdk_bindir / "llvm-"))
             self.use_llvm_binutils = True
 
+        # Support for automatic variable initialization:
+        # See https://github.com/CTSRD-CHERI/cheribsd/commit/57e063b20ec04e543b8a4029871c63bf5cbe6897
+        if self.auto_var_init is AutoVarInit.ZERO:
+            self.make_args.set_with_options(INIT_ALL_ZERO=True)
+        elif self.auto_var_init is AutoVarInit.PATTERN:
+            self.make_args.set_with_options(INIT_ALL_PATTERN=True)
```

Comment:
Do we want to explicitly support None here (e.g. set WITHOUT_INIT_ALL_*=True) in case the default changes in some project or another.

Reply:
That makes sense, will add.